tests/helper/test_helper.py

```diff
@@ -301,3 +306,30 @@ def test_display_error_if_present(mocker, response, footer_updated):
         set_footer_text.assert_called_once_with(response['msg'], 3)
     else:
         set_footer_text.assert_not_called()
+
+
+@pytest.mark.parametrize('media_link, message_id, expected_path', [
+    (SERVER_URL + '/user_uploads/some/path/image.png', 1,
+     gettempdir() + '/zulip-term/1/image.png'),
+    (SERVER_URL + '/user_uploads/some/path/1.png', 1,
+     gettempdir() + '/zulip-term/1/1.png'),
+])
+@pytest.mark.parametrize('platform, expected_command', [
+    ('linux', 'xdg-open'),
+    ('win32', 'explorer'),
+    ('darwin', 'open'),
```

Comment:
I'm not convinced it's worth multiplying the number of tests 3-fold vs having a separate test.

This may also change if we use the 'standard' platforms we use elsewhere now.